packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-button.tsx

```diff
@@ -115,20 +122,25 @@ const CFDCompareAccountsButton = observer(({ trading_platforms, is_demo }: TComp
     );
 
     const onClickAdd = () => {
-        setAppstorePlatform(trading_platforms.platform as string);
-        if (trading_platforms.platform === CFD_PLATFORMS.MT5) {
-            setJurisdictionSelectedShortcode(trading_platforms.shortcode);
-            if (is_account_status_verified) {
-                setAccountType(type_of_account);
-                enableCFDPasswordModal();
+        if (no_real_cr_non_eu_regulator || no_real_mf_account_eu_regulator) {
```

Comment:
How do you handle DIEL countries?

Reply:
This condition handles DIEL countries like either of the real account is not created then show the popup.
Its the same logic as production.

Reply:
#### EU with No real Account
<img width="1450" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/71538efb-9952-422d-9739-059dcc11ea52">

#### Non-EU with Real Account
<img width="1338" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/68898c6e-12d0-4a1a-8d89-d780f46613eb">